lib/constants.js

```diff
@@ -10,3 +10,5 @@ export const SIGNATURE_TYPES = {
 };
 
 export const MAX_ENC_HEADER_LENGTH = 1024;
+
+export const DEFAULT_OFFSET = -86400000;
```

Comment:
I know I said this and Android does the same thing, but maybe if we're going to do this by default, we should default to a more reasonable offset, like 1 minute or 10 minutes or so?

Reply:
Sure, no problem. Since the original problem is in the order of magnitude of seconds, 10 minutes is probably safer than 1?

Reply:
Sure :+1: Even though I feel like if a server is off by a full minute, something has gone pretty wrong (they should all have ntp I think)

Reply:
1 minute it is then!